verification/curator-service/api/test/geocoding/fake.test.ts

```diff
@@ -0,0 +1,39 @@
+import app from '../../src/index';
```

Comment:
you are truly impressive, sir, that you wrote a test for your fake. I applaud you!

Reply:
I am only doing my duty. :)